### PR TITLE
QGIS Raster terrain analysis - tables for alg params

### PR DESCRIPTION
**Pull request by havatv**
Goal:

Ticket(s): #4429

- [x] Backport to LTR documentation is required

### CODE REVIEW

**Comment by DelazJ** [comment]
I don't repeat my comment on each "issue"

File: source/docs/user_manual/processing_algs/qgis/rasterterrainanalysis.rst [hunk 1]
```diff
@@ -33,21 +33,54 @@ The following picture shows the aspect layer reclassified with a color ramp:
...
+     - Description
+   * - **Elevation layer**
+     - ``INPUT``
+     - [raster]
+     - Digital Terrain Model raster layer
+   * - **Z factor**
+     - ``Z-FACTOR``
+     - [number]
+
+       Default: 1
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/rasterterrainanalysis.rst]: That is correct, but the default value, as used when calling the algorithm (you can see it in the log panel) is 1. That is why I have changed them here. I can change them back to include a ".0".

File: source/docs/user_manual/processing_algs/qgis/rasterterrainanalysis.rst [hunk 2]
```diff
@@ -82,81 +115,157 @@ overlap it with the elevation raster:
...
+     - Increasing this value of this parameter will exaggerate the final
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/rasterterrainanalysis.rst]: this ... this was definitely an error.  Will fix / harmonise.

**Comment by havatv** [comment]
Thank you for reviewing, @DelazJ!

**Pull request merged**